archetype-java-builders/src/main/resources/archetype-resources/src/main/java/ExampleJLoadScenarioProvider.java

```diff
@@ -63,7 +66,7 @@ public JLoadScenario exampleJaggerLoadScenario() {
                 .withComment(testDefinitionComment)
                 .withInvoker(ExampleCustomHttpInvokerProvider.nonVerbose())
                 .withQueryProvider(new ExampleQueriesProvider())
-                .withLoadBalancer(new RoundRobinLoadBalancer())
+                .withLoadBalancerProvider(LoadBalancerProvider.ofRandomized(ROUND_ROBIN))
```

Comment:
How will you distinguish randomized and regular load balancer? 
IMO, methods with different names make it more clear for user.